google-cloud-spanner/lib/google/cloud/spanner/client.rb

```diff
@@ -372,7 +372,7 @@ def read table, columns, keys: nil, index: nil, limit: nil,
         # @yield [commit] The block for mutating the data.
         # @yieldparam [Google::Cloud::Spanner::Commit] commit The Commit object.
         #
-        # @return [Boolean] Returns `true` if the operation succeeded.
+        # @return [Time] The timestamp at which the operation committed.
```

Comment:
I would change all of these that read "the operation committed" to "the transaction committed". Or perhaps make it even clearer with the exact API doc from `CommitResponse`: "The Cloud Spanner timestamp at which the transaction committed."